oodeel/extractor/keras_feature_extractor.py

```diff
@@ -166,6 +172,15 @@ def predict_tensor(self, tensor: TensorType) -> Tuple[List[tf.Tensor], tf.Tensor
 
         # split features and logits
         logits = features.pop()
+
+        if postproc_fns is not None:
+            if len(postproc_fns) == 1:
+                features = [postproc_fns[0](features)]
+            else:
+                features = [
+                    postproc_fn(feature)
+                    for feature, postproc_fn in zip(features, postproc_fns)
+                ]
```

Comment:
This will not work if `len(postproc_fns)` is equal to 1 (so in the case of gram, when there is only one `feature_layer_id`) as features is a list. I suggest to remove the lines from 177 to 179 since `features` and `postproc_fns` are of type List in all case:

```suggestion
        if postproc_fns is not None:
            features = [
                postproc_fn(feature)
                for feature, postproc_fn in zip(features, postproc_fns)
            ]
```

Fixing these few lines is however not enough to make Gram work when there is only 1 feature layer id (check my comment on gram.py that raises an other issue about the type of `tensor_stats`).